assets/dev/js/editor/views/base-container.js

```diff
@@ -155,6 +155,28 @@ module.exports = Marionette.CompositeView.extend( {
 		return widget;
 	},
 
+	onDropEvent( event, options ) {
```

Comment:
Why not just like this?
I mean, when you use the `onSomething` convention, others will usually assume it's an event already. What do you think?

```suggestion
	onDrop( event, options ) {
```

Reply:
Done.